experiment/commenter/main_test.go

```diff
@@ -227,7 +227,7 @@ func TestRun(t *testing.T) {
 		{
 			name:    "find none",
 			query:   "none",
-			comment: "this should not happend",
+			comment: "this should not happened",
```

Comment:
s/happened/happen

Reply:
obviously they don't check for grammar errors